### PR TITLE
fixed riak-admin bucket-type create command in examples, capitalized …

### PR DESCRIPTION
**Pull request by korry8911**
…keys

### CODE REVIEW

**Comment by korry8911** [comment]
@mdigan @AleksandrPavlenko

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ The Spark-Riak connector enables you to connect Spark applications to Riak KV an
 
 * Riak TS 1.3.1+
 * Apache Spark 1.6+
-* Scala 2.10
```
Comment (mdigan) [inline README.md]: This release is compatible with 2.10 *and* 2.11, so both should be listed here.

File: examples/README.md [hunk 2]
```diff
@@ -57,7 +57,7 @@ This Scala example demonstrates how to use Riak Spark connector to do range quer
 Before running, you will need to create and activate a TS table called `ts_weather_demo` if it does not already exist. You can find more information on creating and activating TS tables [here](http://docs.basho.com/riak/ts/latest/using/creating-activating/), or you can run the following:
 
 ```
-riak-admin bucket-type create ts_weather_demo '{"props":{"n_val":3, "table_def":"CREATE TABLE ts_weather_demo (weather varchar not null, family varchar not null, time timestamp not null, temperature double, humidity double, pressure double, PRIMARY KEY ((weather, family, quantum(time, 1, 'h')), weather, family, time))"}}'
```
Comment (mdigan) [inline examples/README.md]: Looks like the underlying issue that made my test fail was some non-ASCII characters inside the create table part. If you copy and paste the first version into a file and then use "cat -v", you can see the non-ASCII characters between "temperature" and "double":

```$ cat -v create_ts_weather_demo.txt```

```riak-admin bucket-type create ts_weather_demo '{"props":{"n_val":3, "table_def": "create table ts_weather_demo (weather varchar not null, family varchar not null, time timestamp not null, temperature�M-^@M-^Bdouble, humidity double, pressure double, primary key ((weather, family, quantum(time, 1, 'h')), weather, family, time))"}}'```

However, this formatting fix is fine, so we can keep it.

Comment (korry8911) [inline examples/README.md]: @laurenrother from friday, when we were talking about formatting... when i changed the format to include capitalized keys, its worked, but it was becuase of ^^^^. I guess I deleted the non ascii key. So our discussion of whether the keys are case sensitive is inconclusive. But a quick check could figure this out.

**Comment by korry8911** [comment]
@mdigan nice catch, so it wasnt a case sensitivity issue it seems. fixes made

**Comment by mdigan** [comment]
LGTM 👍

**Pull request merged**